src/main/java/mate/academy/rickandmorty/controller/CharacterController.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.rickandmorty.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.dto.internal.CharacterResponseDto;
+import mate.academy.rickandmorty.service.CharacterService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("api/character")
```

Comment:
```suggestion
@RequestMapping("characters")
```
API is a project wide prefix, better use server.servlet.context-path in properties to set it

Reply:
Thanks for catching that! It looks like my application.properties already had server.servlet.context-path=/api, but I forgot to remove the /api prefix from @RequestMapping. I guess I got a bit too carried away with the double API  @action! I'll fix it right away.